azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/arm/models/Resource.java

```diff
@@ -1,21 +1,21 @@
 /**
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- */
+* Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think the standard format is to have a space: http://www.oracle.com/technetwork/java/codeconventions-141999.html